homeassistant/components/media_player/denonavr.py

```diff
@@ -197,12 +229,27 @@ def source_list(self):
         """Return a list of available input sources."""
         return self._source_list
 
+    @property
+    def sound_mode(self):
+        """Return the current matched sound mode."""
+        return self._sound_mode
+
+    @property
+    def sound_mode_raw(self):
```

Comment:
Same as for **self._sound_mode_raw**. There is no equivalent property in constructor of parent media_player class, thus I assume it is not used.